crates/autopilot/src/arguments.rs

```diff
@@ -130,25 +130,6 @@ pub struct Arguments {
     #[clap(long, env, default_value = "10s", value_parser = humantime::parse_duration)]
     pub auction_update_interval: Duration,
 
-    /// The URL of a list of tokens our settlement contract is willing to
```

Comment:
I'm fine with splitting off these changes into individual PRs when the overall strategy of the solution is agreed upon and signed off on. Until then splitting out these PRs only to redo them 3 more times doesn't seem useful.
That's why I already made them part of this PR so we can see all the smaller changes this would entail in case there are more concerns.